x-pack/plugins/secops/public/components/empty_page/index.test.tsx

```diff
@@ -9,7 +9,7 @@ import toJson from 'enzyme-to-json';
 import React from 'react';
 import { EmptyPage } from './index';
 
-it('renders correctly', () => {
+test('renders correctly', () => {
```

Comment:
Are all these replacements in effort to standardize not using an [alias](https://jestjs.io/docs/en/api.html#testname-fn-timeout)?

Reply:
Yea. Other parts of Kibana use `it(`, but trying to keep it to one set is always nice.